src/Castle.Facilities.WcfIntegration/Service/AbstractServiceHostBuilder.Generic.cs

```diff
@@ -79,7 +79,7 @@ private void ValidateServiceModelInternal(ComponentModel model, TServiceModel se
 			ValidateComponentModel(model);
 			ValidateServiceModel(model, serviceModel);
 
-			var service = model.Services.Single();
+			var service = model.Services.FirstOrDefault();
```

Comment:
Should this use `First` so maintain the requirement for non-null, or is null fine?